odl/util/graphics.py

```diff
@@ -160,6 +160,11 @@ def show_discrete_data(values, grid, title=None, method='',
     axis_fontsize : int, optional
         Fontsize for the axes. Default: 16
 
+    colorbar : bool, optional
+        Argument relevant for 2d plots using ``method`` 'imshow'. If the plot
```

Comment:
2 quick comments:
- \`\`method\`\` 'imshow' -> \`\`method='imshow'\`\`
- "If" can't start a main clause, I guess "Whether" would be more correct. But generally, explanations like "If \`\`True\`\`, include a colorbar in the plot." are preferred.